tests/utils.py

```diff
@@ -567,9 +567,7 @@ def multi_process_parallel(
 
     # Using ray helps debugging the error when it failed
     # as compared to multiprocessing.
-    # NOTE: We need to set working_dir for distributed tests,
-    # otherwise we may get import errors on ray workers
-    ray.init(runtime_env={"working_dir": VLLM_PATH})
+    ray.init()
```

Comment:
Nit: Can you clean this up? I assume you had some issue with setting the runtime_env but it does look like it's necessary.